.github/workflows/zowe-cli-bundle.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
 
     - name: Load Zowe Versions
       id: versions
-      run: node scripts/load-zowe-versions.js ${{ github.event.inputs.package-tag || inputs.package-tag }} ${{ github.event.inputs.release-type || inputs.release-type }}
+      run: node scripts/load-zowe-versions.js ${{ inputs.package-tag }} ${{ inputs.release-type }}
```

Comment:
Thanks for simplifying the workflow 
I had to read up on GH's workflow syntax to be sure of this
_[source](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs)_

Reply:
Yep, fortunately GitHub made an "input unification announcement" since we initially wrote this workflow 🙂 
https://github.blog/changelog/2022-06-10-github-actions-inputs-unified-across-manual-and-reusable-workflows/